packages/api-client/src/api/serializers/cart.ts

```diff
@@ -60,8 +60,9 @@ const deserializeLineItem = (lineItem: any, attachments: JsonApiDocument[], conf
   };
 };
 
-const filterIncludedLineItems = (included: any[]) => {
-  return included.filter(e => e.type === 'line_item');
+const filterIncludedLineItems = (included: any[], apiCart) => {
+  const cartItemIds = apiCart.relationships.line_items.data.map(l => l.id);
+  return included.filter(e => e.type === 'line_item' && cartItemIds.includes(e.id));
```

Comment:
It would be more efficient to create a Set out of ids and check with .has(e.id)

```
  const cartItemIds = new Set(apiCart.relationships.line_items.data.map(l => l.id));
  return included.filter(e => e.type === 'line_item' && cartItemIds.has(e.id));
```

Reply:
Wow, I didn't know that would be faster. Thanks for pointing it out. Gonna go with it next time as I already merged it.